jest.config.js

```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const path = require('path');
 
-const esModules = ['@folio'].join('|');
+const esModules = ['@folio', 'ky'].join('|');
```

Comment:
In fact there are many more ESMs. [jest-config-stripes](https://github.com/folio-org/jest-config-stripes/blob/12e04565402f94eaaa74fb905ecc8dcba3df0773/index.js) has the canonical list. If we can get every repo to point there, then we can actually start upgrading some of those libraries that have converted to ESM in recent releases.